PaddleNLP/paddlenlp/data/sampler.py

```diff
@@ -174,20 +174,37 @@ def batch(self,
               batch_size,
               drop_last=False,
               batch_size_fn=None,
-              batch_fn=None):
+              cmp_fn=None):
         """
         To produce a BatchSampler.
-        Agrs:
+        Args:
             batch_size (int): Batch size.
-            drop_last (bool): Whether to drop the last mini batch. Default: False.
-            batch_size_fn (callable, optional): Return the size of mini batch so far. Default: None.
-            batch_fn (callable, optional): Transformations to be performed. Default: None.
+            drop_last (bool): Whether to drop the last mini batch. Default:
+                False.
+            batch_size_fn (callable, optional): Return the state of mini batch
```

Comment:
`Return the size of mini batch so far.  Actually, the returned value can be anything and would used as argument `size_so_far ` in 'cmp_fn'....`